src/Parser/DemonHunter/Vengeance/Modules/Abilities.js

```diff
@@ -144,14 +146,18 @@ class Abilities extends CoreAbilities {
         spell: SPELLS.SHEAR,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
       },
+      {
+        spell: SPELLS.PAINBRINGER_Buff,
```

Comment:
Wouldn't this cause it to show up in the cast efficiency tab? I think we came to the conclusion that we should just put buffs that doesn't belong to a specific skill in `src/common/DEFENSIVE_BUFFS.js`.

Reply:
True. Forgot about that